R/module_transform_data.R

```diff
@@ -87,7 +87,7 @@ srv_teal_transform_data <- function(id, data, transformators, modules = NULL, is
   names(transformators) <- ids
 
   moduleServer(id, function(input, output, session) {
-    Reduce(
+    module_output <- Reduce(
```

Comment:
@averissimo does this need to be assigned as a variable?

Reply:
It is needed for the `testServer` tests in `test-teal_transform_module.R`

If it's a variable it can be inspected.

Reply:
aaaah got it! I was missing this when I tried to test